### DISCUSSION TITLE
Add Village Onsen Roads

### DISCUSSION THREAD

**Discussion by Smoore1au** [sandermvanvliet/RoadCaptain#113] [Ideas]
Hi Again

with Makuri, i hope/understand you may still be adding in roads for the new Urakazi section.  I have a manual route I've designed which is all the Makuri roads and 88km.  On RoadCaptain though i do not seem to be able to select the perimeter roads around the village Onsen (village gardens and village shops) to complete my route.   

I know it is used in the 'Two Village Loop' official route - am i doing something wrong perhaps ??

**Comment by sandermvanvliet** [comment]
I’d need to take a look into that. My testing on Makuri Islands has been a bit limited so it could be that there are some turns missing that won’t allow you to select these roads. 

I’ll take a look soonish but unfortunately my time is a bit limited at the moment so it may take a while

**Comment by Smoore1au** [comment]
Thanks.  Perhaps take a look at it when or if you get around to adding the new urukazi area roads appreciate your work. If only Zwift would be that progressive thinking !Sent from my iPhoneOn 4 Feb 2023, at 2:38 am, Sander van Vliet ***@***.***> wrote:﻿
I’d need to take a look into that. My testing on Makuri Islands has been a bit limited so it could be that there are some turns missing that won’t allow you to select these roads.
I’ll take a look soonish but unfortunately my time is a bit limited at the moment so it may take a while
